src/main/java/bank/account/Account.java

```diff
@@ -0,0 +1,25 @@
+package bank.account;
+
+import bank.entity.BaseEntity;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EntityListeners;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+@Builder
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "accounts")
+@EntityListeners(AuditingEntityListener.class)
+public class Account extends BaseEntity {
+    private String uid;
+    private String iban;
+    private Integer balance;
+    private Long person_id;
```

Comment:
1. В Java ми так не називаємо змінні: 
```suggestion
    private Long personId;
```
2. Взагалі тут має бути звязок 1- to-many, тож має бути деякі анотації і тоді цей рядок буде мати ось такий вигляд: 
```suggestion
    private Person person;
```